.github/workflows/publish_v2_layer.yml

```diff
@@ -10,10 +10,10 @@ on:
       latest_published_version:
         description: "Latest PyPi published version to rebuild latest docs for, e.g. v2.0.0"
         required: true
-  # workflow_run:
-  #   workflows: ["Publish to PyPi"]
-  #   types:
-  #     - completed
+  workflow_run:
+    workflows: ["Publish to PyPi"]
+    types:
+      - completed
```

Comment:
Instead of `workflow_run`, use a `workflow_call` and call it explicitly after the "Publish to Pypi" action